<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PracticeProject2Class</class>
 <widget class="QMainWindow" name="PracticeProject2Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PracticeProject2</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTableView" name="starstableView"/>
    </item>
    <item>
     <widget class="QPushButton" name="viewStarpushButton">
      <property name="text">
       <string>View</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox">
      <property name="text">
       <string>CheckBox</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
    <item>
     <widget class="QListWidget" name="listWidget"/>
    </item>
    <item>
     <widget class="QPushButton" name="addStarpushButton">
      <property name="text">
       <string>Add Star</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="addStarlineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="PracticeProject2.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>checkBox</sender>
   <signal>stateChanged(int)</signal>
   <receiver>PracticeProject2Class</receiver>
   <slot>starsCheckBox()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>169</x>
     <y>183</y>
    </hint>
    <hint type="destinationlabel">
     <x>3</x>
     <y>125</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addStarpushButton</sender>
   <signal>clicked()</signal>
   <receiver>PracticeProject2Class</receiver>
   <slot>addStar()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>61</x>
     <y>326</y>
    </hint>
    <hint type="destinationlabel">
     <x>3</x>
     <y>322</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>PracticeProject2Class</receiver>
   <slot>onTheGo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>556</x>
     <y>205</y>
    </hint>
    <hint type="destinationlabel">
     <x>598</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>viewStarpushButton</sender>
   <signal>clicked()</signal>
   <receiver>PracticeProject2Class</receiver>
   <slot>viewStar()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>42</x>
     <y>148</y>
    </hint>
    <hint type="destinationlabel">
     <x>7</x>
     <y>77</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>starsCheckBox()</slot>
  <slot>addStar()</slot>
  <slot>onTheGo()</slot>
  <slot>viewStar()</slot>
 </slots>
</ui>
